set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -83,7 +83,17 @@ public UserSet getUserSetById(String userSetId) throws UserSetNotFoundException
 	return res;
     }
 
-    /**
+	@Override
+	public List<PersistentUserSet> getUserSetByCreatorId(String creatorId) throws UserSetNotFoundException {
+		List<PersistentUserSet> res = getMongoPersistence().getByCreator(creatorId).asList();
+		if (res.isEmpty()) {
+			throw new UserSetNotFoundException(UserSetI18nConstants.USERSET_NOT_FOUND_FOR_USER,
```

Comment:
no exception should be thrown if there are no sets created by an user

Reply:
I will remove the exception